### DISCUSSION TITLE
[Solved] Add Fiber Interface Type E2000

### DISCUSSION THREAD

**Discussion by Daniel-Dietz** [netbox-community/netbox#8690] [General]
Hi,

I do need an Interface E2000 used by carriers for Fiber lines 10GB+ 
LWL-Type is defined by IEC 61754-15 and CECC 86275

I would suggest not to deep dive into the fiber angles, there are multiple identified by a color coding of the fiber jack.

**Comment by julianze** [comment]
E2000 is a LSH connector and already included: #2708

**Reply by Daniel-Dietz** [reply]
Thank you – I found this as well a few minutes ago. Thank you for the help :-)
